guides/release/code-editors/index.md

```diff
@@ -71,6 +80,9 @@ Atom autocomplete and snippets for Ember.js.
 [atom-ember-snippets](https://github.com/mattmcmanus/atom-ember-snippets) -
 Ember.js ES6, Ember Data & Handlebars snippets for Atom editor.
 
+[language-ember-htmlbars](https://atom.io/packages/language-ember-htmlbars) - 
```

Comment:
By default, Atom uses `HTML (handlebars)`, which looks absolutely terrible.
This plugin isn't great either (last update was 2017), but it's better than _nothing_.

I found this while trying to figure out how to update GitHub's highlighting... but.. that's _[been a ride](https://github.com/github/linguist/pull/5631) so far